src/neo/Plugins/Plugin.cs

```diff
@@ -46,15 +46,15 @@ static Plugin()
 
         protected Plugin()
         {
+            Configure();
+            
             Plugins.Add(this);
 
             if (this is ILogPlugin logger) Loggers.Add(logger);
             if (this is IStoragePlugin storage) Storages.Add(Name, storage);
             if (this is IP2PPlugin p2p) P2PPlugins.Add(p2p);
             if (this is IPersistencePlugin persistence) PersistencePlugins.Add(persistence);
             if (this is IMemoryPoolTxObserverPlugin txObserver) TxObserverPlugins.Add(txObserver);
-
-            Configure();
```

Comment:
Any help with moving `Configure()` here?

Reply:
There might be exception in when Configure. I think we should add the plugin after it configure successfully.

Reply:
I prefer to remove the log in `Configure()`.

Reply:
In this case users won't notice the json file format error.
What about not catching in `Utility.LoadConfig` and throwing this exception to high level.
Although the exception will be catched in `Plugin.LoadPlugin` but will be throwed when blockchain initializes. In this case Plugins load failed and Blockchain throws exception. Users can't start neo.

Reply:
So this PR can be closed?

Reply:
yes, I think. Merge and close.

Reply:
There is nothing to be merged.